### PR TITLE
Awesome books - Third milestone

### PR DESCRIPTION
**Pull request by utsargo**
### Awesome books - Third milestone

**Changes in this milestone**

- Add a header with a menu
- Add footer
- Implement switching content using javascript

**In this attempt:**

- The previous reviewer asked me to remove eslint disable rules. I kept only which are absolutely necessary for my approach.
- Removes unnecessary comments from the javascript file.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 /* eslint-disable no-use-before-define */
 /* eslint-disable max-classes-per-file */
 /* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
```
Comment (omarsalem7) [inline index.js]: - [ ] Kindly avoid disabling linter errors and fix them to make your code professional and clean.

File: index.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
 // Create A constructor class for book object. Object should have:
 //  1. Book name/title
 //  2. Book author
 //  3. A unique id to identify the book (we can use Math.random() to generate the id)
```
Comment (omarsalem7) [inline index.js]: - [ ] Kindly remove these unneeded lines to make your code clean.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi team 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (utsargo) [reply index.js]: @omarsalem7 I've disabled three eslint rules which are necessary for this code. I agree that disabling the no-plus-plus rule is unnecessary. But the other three are needed for my approach.

**Review by Olamarx — APPROVED** [review]
Hi @utsargo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linters passed ✔️ 
- Readme file ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**